code/src/index.css

```diff
@@ -1,13 +1,195 @@
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  margin: 10;
+  font-family: 'Roboto Mono', monospace;
+  font-weight: 500;
+  background-color: rgb(255, 220, 226);
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+main {
+  display: flex;
+  justify-content: center;
+  /* flex-wrap: wrap; */
+  margin: 10px 0;
+}
+
+.content-wrapper {
+  width: 450px;
+}
+
+.form-wrapper {
+  background-color: rgb(242, 240, 240);
+  border: 2px solid black;
+  box-shadow: 6px 8px;
+  margin: 30px 15px;
+  line-height: 1rem;
+  padding: 10px 15px 5px;
+  display: flex;
+  flex-direction: column;
+}
+
+.form-input {
+  border: 2px solid rgb(191, 191, 191);
+  margin-top: 12px;
+  height: 50px;
+  padding: 10px;
+  font-family: 'Roboto Mono', monospace;
+  /* font-size: 10px; */
+}
+
+.count-text {
+  font-size: 10px;
+}
+
+.count-text-error {
+  font-size: 10px;
+  color: red;
+}
+
+.thought-text {
+  font-size: 12px;
+  letter-spacing: -0.5px;
+  /* margin-bottom: 12px; */
+}
+
+.thought-wrapper {
+  background-color: white;
+  border: 2px solid black;
+  box-shadow: 6px 6px;
+  margin: 30px 15px;
+  line-height: 1rem;
+  padding: 0 15px;
+}
+
+.form-btn {
+  width: 50%;
+  padding: 10px;
+  margin: 10px 0;
+  border: none;
+  border-radius: 20px;
+  background-color: rgb(255, 173, 173);
+  font-size: 12px;
+  font-weight: bold;
+  color: rgb(78, 52, 52);
+  cursor: pointer;
+}
+
+.form-btn:hover {
+  width: 52%;
+  font-size: 13px;
+}
+
+.message-text {
+  font-size: 14px;
+  overflow-wrap: break-word;
+  line-height: 2em;
+}
+
+.like-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  color: rgba(61, 61, 61, 0.8);
+  padding: 5px 0;
+  /* padding-bottom: 5px; */
+  margin-bottom: 8px;
+}
+
+.like-btn {
+  display: flex;
+  letter-spacing: 0.2em;
+  align-items: center;
+  color: rgb(162, 162, 162);
+  letter-spacing: -2px;
+  font-size: 12px;
+}
+
+.heart-btn {
+  width: 40px;
+  height: 40px;
+  border-radius: 100%;
+  border: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-right: 10px;
+  background-color: rgb(234, 234, 234);
+  cursor: pointer;
+}
+
+.heart-btn-clicked {
+  background-color: rgb(255, 173, 173);
+}
+
+.heart-btn:hover {
+  width: 43px;
+  height: 43px;
+}
```

Comment:
To avoid making the text-box getting higher when hovering on the button, you could have removed as much margin from the button as the button grows.